CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 Changelogs for versions not listed here can be found at https://github.com/DataDog/dd-trace-py/releases
 
+---
+
+## 2.16.7
+
+
+### Bug Fixes
+
+  - Updates setup.py to ignore int-ptr conversion warnings for the profiler stack.pyx file. This is important because gcc 14 makes these conversions an error, alpine 3.21.0 ships with gcc 14, and any patch version of a Python alpine image cut after December 5th, 2024, will have this issue.
```

Comment:
I'm not 100% sure this should be outward-facing.  I had to mark this patch as a bug fix in order for it to satisfy our backport policy, but really it only generally mattered for CI.

I guess it doesn't hurt to raise awareness of this issue.  :shrug: